x-pack/legacy/plugins/siem/server/lib/detection_engine/README.md

```diff
@@ -1,6 +1,45 @@
-Temporary README.md for developers working on the backend detection engine
+Temporary README.md for users and developers working on the backend detection engine
 for how to get started.
 
+# Setup for Users
+
+If you're just a user and want to enable the REST interfaces and UI screens do the following.
+NOTE: this is very temporary and once alerting and actions is enabled by default you will no
+longer have to do these steps
+
+Set the environment variable ALERTING_FEATURE_ENABLED to be true in your .profile or your windows
+global environment variable.
+
+```sh
+export ALERTING_FEATURE_ENABLED=true
+```
+
+In your `kibana.yml` file enable alerting and actions like so:
+
+```sh
+# Feature flag to turn on alerting
+# xpack.alerting.enabled: true
+
+# Feature flag to turn on actions which goes with alerting
+# xpack.actions.enabled: true
+```
```

Comment:
I feel people will just copy and paste this right into their kibana.yml without uncommenting the properties? Might be a better overall experience if we uncomment the properties in the doc.

```
# Feature flag to turn on alerting
xpack.alerting.enabled: true

# Feature flag to turn on actions which goes with alerting
pack.actions.enabled: true
```

Reply:
Agreed. Also found an issue with Ben Skelker's environment where you have to run without base path or you end up having to change all of your POST/GET calls which I think is a feature but gets in the way of getting people setup.